antiweb.py

```diff
@@ -2825,21 +2889,112 @@ def write(path, fname, output, token, warnings, index, index_rst, recursive, con
 
 #@edoc
 
-
 #If the user added the -i flag, the file gets added to Sphinx' index.rst file. Between the :py:class:`start(generated)` and :py:class:`end(generated)` directives is the space for automatic added files, you can manually add files below the :py:class:`end(generated)` directive.
 
 #@code
 
         if index:
             if could_process:
-                
                 if output and recursive:
-                    replace_in_generated(start_of_block, end_of_block, out_file_name, path, output, index_rst, content, startline)
+                    replace_in_generated(start_of_block, end_of_block, out_file_name_index, path, output, index_rst, content, startline)
                 else:
-                    replace_in_generated(start_of_block, end_of_block, out_file_name, path, None, index_rst, content, startline)
+                    replace_in_generated(start_of_block, end_of_block, out_file_name_index, path, None, index_rst, content, startline)
 #@edoc
-
 #@(write)
+
+#@cstart(get_output_file_name)
+
+def get_output_file_name(working_dir, input_file_name, output, directory, file):
+#@start(get_output_file_name doc)
+    """
+.. py:method:: get_output_file_name(working_dir, input_file_name, output, directory, file)
+    
+   Computes the absolute path to the output file based on the output commandline option.
+           
+   :param working_dir: Current working directory.
+   :param input_file_name: The name of the file which should be processed.
+   :param output: The given -output parameter.
+   :param directory: The dedicated report directory.
+   :param file: The file part from the -output parameter.
```

Comment:
Why do you need it as a separate parameter? Can you compute it internally from the *output* parameter? !resolve